### ISSUE TITLE
Add visual cascades option to indentation

### ISSUE THREAD

**Issue by silverwind** [stylelint/stylelint#628]
The following should not be considered a error with an option which could be called `visualCascade` or possibly `alignAtColon`:
```css
body {
    -webkit-filter: invert(100%);
            filter: invert(100%);
}
```

**Comment by davidtheclark** [comment]
I understand the request. Can you describe any precise logic that would differentiate stuff like this from an error? That would be helpful. Or a PR.

It gets even more complicated if you want to enforce something like this:

```css
// Bad
   -moz-filter: invert(100%);
-webkit-filter: invert(100%);
        filter: invert(100%);

// Bad
-webkit-filter: invert(100%);
   -moz-filter: invert(100%);
        filter: invert(100%);
```

... which you probably do?

**Comment by silverwind** [comment]
Maybe `indentPrefixedAtColon` would even be a better name. It could go like this:

1) Search for rules that start with a `-` and exempt them from regular indentation rules
2) For each rule found, check if there's a unprefixed rule, and if found, save the indentation level of `:`
3) For each prefixed rule, calculate the required indentation level based on the unprefixed rule
4) Pass the `indentPrefixedAtColon` rule If the required indenation matches, else fail

Ordering from longest to shortest would be a nice bonus, but I don't really see it as absolutely necessary, because maybe someone prefers it the other way around?

**Comment by davidtheclark** [comment]
Cool, thanks for thinking through a possible approach, @silverwind. I'll label this `pr-welcome`.

**Comment by jeddy3** [comment]
Just doing some issue triage...

Do we need this option in the age of `autoprefixer`? All of the other rules [generally assume](https://github.com/stylelint/stylelint/blob/master/src/rules/function-blacklist/README.md#options) the user will be working with unprefixed code. We also have a couple of rules that specifically check and warn for prefixes, and no rules that encourage having them.

So, I'm not sure it is worth adding complexity to the already complex `indentation` rule. @silverwind [prefers autofixing over warnings](https://github.com/stylelint/stylelint/issues/669#issuecomment-177135319), and so we may no longer have a solid use case for this. I suggest we close this issue and see if it comes up again before taking any action.

@davidtheclark What do you think?

**Comment by davidtheclark** [comment]
Sure. I ha don intention to implement it myself, just left it in case anyone wanted to pick it up. But that's very unlikely. Let's close.
